fluent-plugin-protobuf/lib/fluent/plugin/parser_protobuf.rb

```diff
@@ -11,17 +10,15 @@ module Plugin
     class ProtobufParser < Fluent::Plugin::Parser
       Fluent::Plugin.register_parser('protobuf', self)
 
+      KEY_TIMESERIES = 'timeseries'.freeze
+
       def parse(text)
-        begin
-          inflated = Snappy.inflate(text)
-          decoded = Prometheus::WriteRequest.decode(inflated)
-          decoded.timeseries.map { |ts|
-            log.debug(ts)
-            ts
-          }
-        rescue StandardError => exception
-          log.error('ERROR during decoding', error: exception)
-        end
+        inflated = Snappy.inflate(text)
+        decoded = Prometheus::WriteRequest.decode(inflated)
+        log.trace "protobuf::parse - in: (#{text.bytesize}/#{inflated.bytesize}), out: #{decoded.timeseries.length}"
+        record = {}
+        record[KEY_TIMESERIES] = decoded.timeseries
+        yield Fluent::EventTime.now, record
```

Comment:
Do we override the time eventually using the value in record itself?

Reply:
yes. this time is only used in fluentd pipeline